test/e2e/scheduling/preemption.go

```diff
@@ -467,6 +468,71 @@ var _ = SIGDescribe("PreemptionExecutionPath", func() {
 			}
 		}
 	})
+	// this test makes sure that Pods with a higher PriorityClass via their PriorityClassName start earlier than Pods with a lower PriorityClassName
```

Comment:
This statement doesn't look right. PriorityClass only takes into effect when the incoming pod can't fit on any node - when the so-called "preemption" happens.

Reply:
https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/